cluster-autoscaler/cloudprovider/gce/gce_cloud_provider.go

```diff
@@ -230,6 +231,11 @@ func (mig *Mig) Nodes() ([]string, error) {
 	return mig.gceManager.GetMigNodes(mig)
 }
 
+// TemplateNodeInfo returns a node template for this node group.
+func (mig *Mig) TemplateNodeInfo() (*schedulercache.NodeInfo, error) {
+	return nil, fmt.Errorf("TemplateNodeInfo not supported")
```

Comment:
I assume we allow for this to be not implemented for some cloud providers? I think that 'not implemented' is a very different situation than this method returning an error and we should be able to distinguish the two easily (not just by looking for hardcoded string here).